deploy/self-hosted/.env.example

```
@@ -147,7 +147,7 @@ REVIEW_ROUTER_REVIEW_INVESTIGATION_SELECTORS_JSON=
 # Provider credentials stay in customer GitHub Actions secrets, never here.
 REVIEW_ROUTER_ENABLE_CODEX_ROTATING_OAUTH=1
 REVIEW_ROUTER_CODEX_ROTATING_OAUTH_REPOSITORIES=
-REVIEW_ROUTER_DEFAULT_MODEL=gpt-5.5
+REVIEW_ROUTER_DEFAULT_MODEL=gpt-5.6-sol
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REVIEW_ROUTER_DEFAULT_MODEL key should go before the REVIEW_ROUTER_ENABLE_CODEX_ROTATING_OAUTH key

(UnorderedKey)